### PR TITLE
feat: button group helper

### PR DESCRIPTION
**Pull request by n1ru4l**
image

### CODE REVIEW

File: packages/leva/src/components/ButtonGroup/StyledButtonGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { styled } from '../../styles'
+
+export const StyledButtonGroup = styled('div', {
+  $leva__flex: '',
+  display: 'flex',
```
Comment (dbismut) [inline packages/leva/src/components/ButtonGroup/StyledButtonGroup.tsx:5]: `$leva_flex: ''` already applies the flex display so you don't need that line.

File: packages/leva/src/components/ButtonGroup/StyledButtonGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  justifyContent: 'flex-end',
+  gap: '5px',
```
Comment (dbismut) [inline packages/leva/src/components/ButtonGroup/StyledButtonGroup.tsx:7]: There's a `$leva__rowGap` stitches property that you can use here for consistency.

```js
gap: '$leva__rowGap'
```

**Review by dbismut — CHANGES_REQUESTED** [review]

File: packages/leva/stories/inputs/Button.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+export default {
+  title: 'Inputs/Button',
+  decorators: [Reset],
+} as Meta
+
+const Template: Story<any> = (args) => {
+  const values = useControls({
+    foo: button(() => alert('click')),
+    bars: buttonGroup({
```
Comment (dbismut) [inline packages/leva/stories/inputs/Button.stories.tsx:16]: make this a separate story

File: packages/leva/src/components/ButtonGroup/ButtonGroup.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  }
+}
+
+export function ButtonGroup({ label, opts }: ButtonGroupProps) {
+  return (
+    <Row input={true}>
+      <Label>{label}</Label>
+      <StyledButtonGroup>
+        {Object.entries(opts).map(([label, onClick]) => (
+          <Button onClick={() => onClick()} label={label} />
```
Comment (dbismut) [inline packages/leva/src/components/ButtonGroup/ButtonGroup.tsx:19]: React complains about missing `keys`

**Review by dbismut — CHANGES_REQUESTED** [review]

File: packages/leva/src/components/ButtonGroup/StyledButtonGroup.tsx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import { styled } from '../../styles'
+
+export const StyledButtonGroup = styled('div', {
+  $leva__flex: '',
+  justifyContent: 'flex-end',
+  gap: '$leva__rowGap',
```
Comment (dbismut) [inline packages/leva/src/components/ButtonGroup/StyledButtonGroup.tsx:6]: Sorry I said `rowGap` It should be `colGap` I'm an idiot 😅

Comment (n1ru4l) [reply packages/leva/src/components/ButtonGroup/StyledButtonGroup.tsx]: done :)

**Review by dbismut — APPROVED** [review]

**Pull request merged**